### ISSUE TITLE
Visual Code Azure error

### ISSUE THREAD

**Issue by n8te** [Microsoft/vscode-azureappservice#611]
&lt;Please be sure to remove any private information before submitting.&gt;

Repro steps:
&lt;Enter steps to reproduce issue&gt;

Action: appService.showFile
Error type: Error
Error Message: TextEditor(vs.editor.ICodeEditor:1,$model30) has been disposed

Version: 0.9.1
OS: win32

**Comment by StephenWeatherford** [comment]
https://github.com/Microsoft/vscode-cosmosdb/issues/681

**Comment by nturinski** [comment]
Yeah, this seems to happen when you click on a file multiple times.  Seems like VS Code can't handle opening that many files in quick succession?

The uploading is happening because I have "auto-save" enabled so when it closes, I'm assuming it's trying to save it.

![image](https://user-images.githubusercontent.com/5290572/48577968-53ff9a80-e8cd-11e8-92a7-a35567a20fd9.png)
